src/support/ssz.rs

```diff
@@ -0,0 +1,66 @@
+#![cfg(feature = "ssz")]
+#![cfg_attr(docsrs, doc(cfg(feature = "ssz")))]
+use ssz::{Decode, DecodeError, Encode};
+
+use crate::{nbytes, Uint};
+
+impl<const BITS: usize, const LIMBS: usize> Encode for Uint<BITS, LIMBS> {
+    fn is_ssz_fixed_len() -> bool {
+        true
+    }
+
+    fn ssz_fixed_len() -> usize {
+        nbytes(BITS)
+    }
+
+    fn ssz_bytes_len(&self) -> usize {
+        nbytes(BITS)
+    }
+
+    fn ssz_append(&self, buf: &mut Vec<u8>) {
+        buf.extend_from_slice(&self.as_le_bytes());
+    }
+}
+
+impl<const BITS: usize, const LIMBS: usize> Decode for Uint<BITS, LIMBS> {
+    fn is_ssz_fixed_len() -> bool {
+        true
+    }
+
+    fn ssz_fixed_len() -> usize {
+        nbytes(BITS)
+    }
+
+    fn from_ssz_bytes(bytes: &[u8]) -> Result<Self, DecodeError> {
+        if bytes.len() > nbytes(BITS) {
+            return Err(DecodeError::InvalidByteLength {
+                len:      bytes.len(),
+                expected: nbytes(BITS),
+            });
+        }
+        let mut buf = [0u8; nbytes(BITS)]; // generic parameters may not be used in const operations
```

Comment:
Having some issue on using a generic parameter as a constant here.

Any advice @prestwich or @DaniPopes ?

Reply:
You should be able to do `Ok(Self::from_le_slice(bytes))`, no?